src/opentelemetry/ext/lightstep/__init__.py

```diff
@@ -57,6 +57,11 @@ def export(self, spans: typing.Sequence[Span]) -> "SpanExportResult":
                 start_time=_nsec_to_sec(span.start_time),
                 tags=span.attributes,
             )
+            for event in span.events:
+                event.attributes["message"] = event.name
```

Comment:
Just a bit of semantic confusion here, the "message" of the event is also its "name"?

Reply:
Yeah, this is setting the `message` attribute to the name, as logs don't really have the concept of a name